.github/ISSUE_TEMPLATE/operate_bug_report.md

```diff
@@ -8,67 +8,38 @@ assignees: ''
 
 ---
 
-### Describe the bug
+### Description (required on creation)
 
 <!-- A clear and concise description of what the problem/bug is about and what is the current behavior. -->
 
-### To Reproduce
+### Steps to reproduce (required on creation)
 
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
+<!-- Make a list of steps here -->
 
-### Current behavior
+### Current behavior (required on creation)
 
-<!-- Please attach screenshots, a screen recording, or a file (e.g. the BPMN/DMN/Form file) that has the problem you are describing to help us better debug the respective issue. -->
+<!-- Describe the erroneous behavior after the reproduction steps are applied. Attach screenshots, a screen recording, or other files that can help. -->
 
-### Expected behavior
+### Expected behavior (required on creation)
+<!-- In comparison to the current behavior, describe what should happen instead for the product to work correctly -->
 
-<!-- A clear and concise description of what you expected to happen. -->
-
-### Environment
+### Environment (required on creation)
```

Comment:
I rarely used this section. What value does it have?

Reply:
It's particularly helpful if a bug only applies in a certain setup. E.g. only with Opensearch. Only on 8.5. Only on a particular browser version. Having it as a dedicated section is a reminder to provide that information. For the majority of tickets it's irrelevant though because they apply in any setup. So we could also consider removing that section if everyone is generally aware that this information is important in some cases. It can also help with community tickets where people otherwise wouldn't share that.

Reply:
Yup, will keep it.